docs/source/eogrow.core.area.rst

```diff
@@ -0,0 +1,16 @@
+eogrow.core.area
```

Comment:
Sphinx build is telling me that this file is not included in toctree.

Reply:
fixed